src/components/Admin/index.jsx

```diff
@@ -59,17 +63,17 @@ class Admin extends React.Component {
       active: {
         title: 'Learners Enrolled in a Course',
         subtitle: 'Top Active Learners',
-        component: <EnrollmentsTable />,
+        component: <ActiveLearnersTable />,
       },
       'inactive-week': {
         title: 'Learners Enrolled in a Course',
         subtitle: 'Not Active in Past Week',
-        component: <EnrollmentsTable />,
+        component: <InactivePastWeekLearnersTable />,
```

Comment:
Because the `InactivePastWeekLearnersTable` and `InactivePastMonthLearnersTable` share the same columns, I wonder if we might want to make a single component `InactiveLearnersTable` that takes in props to determine which version (e.g., past week or month) to use. The table could default to fetching past week's inactive learners, and have a boolean prop for `pastMonth`. For example,

```javascript
{
  'inactive-week': {
    ...,
    component: <InactiveLearnersTable />, // fetch inactive learners in past week by default
  },
  'inactive-month': {
    ...,
    component: <InactiveLearnersTable pastMonth />, // fetch inactive learners in past month due to pastMonth prop
  },
}
```

If we do this, the `InactiveLearnersTable` component would simply use the appropriate `fetchMethod` supplied to `TableContainer` based on the `pastMonth` prop, e.g.:

```javascript
getFetchMethod() {
  if (this.props.pastMonth) {
    return EnterpriseDataApiService.fetchInactivePastMonthLearners;
  }
  return  EnterpriseDataApiService.fetchInactivePastWeekLearners;
}
```

```jsx
<TableContainer
  ...
  fetchMethod: this.getFetchMethod(),
/>
```

Reply:
So I guess, active and inactive past week and past Month can just be merged into componen, with different fetch method. I will let you know Tommorow when I address this feedback ✌️

Reply:
@mushtaqak The inactive past week/month can be merged into single component but not the active week table. the columns for those tables are not exactly the same as the active learners per the ticket. The inactive week/month tables don't show any data related to passing (no "Passed Date" column).

Though, I suppose you could have all three tables derived from the same component and modify the columns based on the prop passed in.